File: app/src/main/java/intellij/class5/objetos/exercise5/java/Concessionaria.java
package intellij.class5.objetos.exercise5.java;

public class Concessionaria {
    public void registrarVenda(Veiculo veiculo, Cliente cliente, Double valor) {
        String nomeDoVeiculo = veiculo.getMarca() + " " + veiculo.getModelo() + " " + veiculo.getAnoDeFabricacao();
        String nomeDoCliente = cliente.getNome() + " " + cliente.getSobrenome();

        Venda venda = new Venda(cliente, veiculo, valor);

        System.out.println("Veículo vendido: " + nomeDoVeiculo + "\nPara: " + nomeDoCliente + "\nPelo valor de: R$ " + valor);
    }
}

File: app/src/main/java/intellij/class5/objetos/exercise5/kotlin/Concessionaria.kt
package intellij.class5.objetos.exercise5.kotlin

class Concessionaria {
    fun registrarVenda(veiculo: Veiculo, cliente: Cliente, valor: Double) {
        val nomeDoVeiculo = veiculo.marca + " " + veiculo.modelo + " " + veiculo.anoDeFabricacao
        val nomeDoCliente = cliente.nome + " " + cliente.sobrenome

        val venda = Venda(cliente, veiculo, valor)

        println("Veículo vendido: $nomeDoVeiculo\nPara: $nomeDoCliente\nPelo valor de: R$ $valor.")
    }
}